app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -420,8 +422,17 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), UserInteractionHan
         return {
             deleteSelectedBookmarks(pendingBookmarksToDelete)
             pendingBookmarkDeletionJob = null
-            // Since this runs in a coroutine, we can't depend upon the fragment still being attached
-            metrics?.track(event)
+            when (event) {
+                is Event.RemoveBookmarkFolder ->
+                    BookmarksManagement.folderRemove.record(NoExtras())
+                is Event.RemoveBookmarks ->
+                    BookmarksManagement.multiRemoved.record(NoExtras())
+                is Event.RemoveBookmark ->
+                    BookmarksManagement.removed.record(NoExtras())
+                else -> {
+                    // no-op, no other bookmark delete event types
```

Comment:
Should we log this, send a caught exception report or throw an exception?  (In case we missed something or something new is added)